src/IgnoreList.cxx

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The Music Player Daemon Project
+
+#include "IgnoreList.hxx"
+
+#include <cassert>
+#include <ranges>
+#include <algorithm>
+
+bool IgnoreListEntry::matches_record(const Record& record) const {
```

Comment:
Bad coding style, and you forgot `noexcept`

Reply:
Apologies, will fix.